xbmc/music/dialogs/GUIDialogVisualisationPresetList.cpp

```diff
@@ -28,8 +28,8 @@
 #include "utils/StringUtils.h"
 
 #define CONTROL_HEADER_LABEL   1
-#define CONTROL_NONE_AVAILABLE 4
```

Comment:
I see. Thats what you get for not looking in Confluence. I (and maybe more skinners) didnt have label with id 4 in Nox. It was not mentioned in #7863 or in the skin changes thread. Will revert, thx.